### DISCUSSION TITLE
欢迎来到52ABP社区讨论！/ Welcome to 52ABP CommunityDiscussions!

### DISCUSSION THREAD

**Discussion by ltm0203** [52ABP/LTMCompanyNameFree.YoyoCmsTemplate#55] [Announcements 公告]
## 👋 欢迎!

我们使用讨论作为与我们社区其他成员联系的地方。我们希望你

- 提出你想知道的问题。
- 分享想法。
- 与其他社区成员互动。
- 欢迎他人并保持开放的心态。记住，这是一个我们一起建立的💪。
要想开始，请在下面评论中介绍自己，并告诉我们你在这个社区做了什么。

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JanFrank** [comment]
yoyo,new discussions

**Reply by ltm0203** [reply]
是的，以后就有主阵地了。挺好了。

**Reply by JanFrank** [reply]
缺少表情包了，有个表情包，又是水军大战之地

**Reply by ltm0203** [reply]
> 缺少表情包了，有个表情包，又是水军大战之地

还是主技术吧。灌水太多 没意思了。

**Reply by JanFrank** [reply]
大佬加油，我等只能围观了
